### ISSUE TITLE
Does not accept versions like v0.12.0

### ISSUE THREAD

**Issue by RReverser** [aaronpowell/ps-nvmw#1]
Getting following error:

```
Cannot validate argument on parameter 'Version'. The argument "v0.12.0" does not match the "^v\d\.\d{2}\.\d{2}$" pattern. Supply an argument that matches
```

While this version is completely correct, and using `v0.12.00` instead returns 404.

**Comment by aaronpowell** [comment]
Whoops, bad regex. Fixed with 9b02d516d2130e7551041063957e2c1367f3ab07

**Comment by RReverser** [comment]
@aaronpowell Thanks!
